scylla-gdb.py

```diff
@@ -3789,6 +3789,25 @@ def invoke(self, arg, for_tty):
             f = reference_wrapper(f).get()
```

Comment:
Please add an example printout to the commit log.

Reply:
ok.